checklists/creating-interactive-prototypes.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Creating Interactive Prototypes
+category: Design
+date: "2020-03-11"
+tags: ['tools', 'prototyping', 'design-process']
+description: Creating interactive prototypes is an important step in the product design process. Use this checklist to make sure you cover the basics of creating them!
+---
+
+- [ ] Create the screens  
+These screens don't have to be high-fidelity designs.
+
+- [ ] Wire them up  
+Make sure you have linked every screen to each other so that ther are no gaps in interactions.
+
+- [ ] Pay attention to component behaviours  
+For better prototyping you should pay extra attention to component states and their behaviours. E.g. sticky components, hover and active effects, micro-interactions...
```

Comment:
You can be more explicit here. I think you can divide the examples to separated to-do's

Reply:
Reason why I am not specifying those into this checklist is that I am planning on to create another checklist that covers the essentials of the prototyping processes. This checklist here is more general than the one I will create soon. 

Is that a bad idea? @alpcanaydin